src/createServer.js

```diff
@@ -1,3 +1,74 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const url = require('url');
+const { convertToCase } = require('./convertToCase');
+
+const MISSING_TEXT =
+  // eslint-disable-next-line max-len
+  'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+const MISSING_CASE =
+  // eslint-disable-next-line max-len
+  '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+const UNSUPPORTED_CASE =
+  // eslint-disable-next-line max-len
+  'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.';
```

Comment:
Generate the message with "Available cases" depending on `SUPPORTED_CASES ` variable